lib/cppcheck.cpp

```diff
@@ -659,6 +660,7 @@ unsigned int CppCheck::checkFile(const std::string& filename, const std::string
                     reportErr(errmsg);
                 }
             }
+            std::remove(dumpFile.c_str());
```

Comment:
Does this work on Windows? As I remember, there was some problems with `std::remove` with Windows on AppVeyor build, so I had to [use](https://github.com/danmar/cppcheck/pull/2367/files#diff-e2c07bae40af8b7fa345b5722aa544f6R103) [DeleteFile](https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-deletefilea) function from Win32 API.
Just to clarify, never worked with not *nix systems.

Reply:
I checked it with small sketch on Win10 and VS2017 and it working in such environment.